### DISCUSSION TITLE
Handling when a merged branch can't be safely deleted

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#73300] [Pull Requests]
### Select Topic Area

General

### Body

Normally if you create a PR and the PR is merged, GitHub will show a message saying that the branch can be safely deleted and offering a button to delete the branch.

Of late, I've hit a number of cases where that button has been missing.

It's really confusing.

Technically, GitHub has done enough work to identify either:
* that the branch is the default branch (which presumably can't be deleted); or
* that the branch is connected to another PR (definitely as the PR Head, but I half imagine that the same would apply if it was the PR base)

Given that I have to presume that GitHub has figured out which of these apply, I'd be much happier if GitHub could tell me:
* this branch is the default branch and thus can't be deleted; or
* this branch is the PR Head for at least _one other PR_ (with that being a link to that PR, and thus can't be deleted

I have at least 3 repositories in almost all of my fork networks, and can easily have more, so when I discover that a PR head can't be deleted, it is't remotely obvious where I should go to look to find the other PR for which it's a Head....

**Comment by Nouri11190** [comment]
What should I do to pull out my project on GitHub this is getting months now this keep delaying what is the problem 
Oh my gosh I am really (tired) I need a help to take my tokens trapped here in GitHub
